src/autogluon/bench/frameworks/tabular/exec.sh

```diff
@@ -1,33 +1,32 @@
 #!/bin/bash
 
-framework=${1:-"AutoGluon:latest"}
-benchmark=${2:-"test"}
-constraint=${3:-"test"}
-DIR=${4:-"./benchmark_runs/tabular/test"}  # from root of project
+framework=${1}
+benchmark=${2}
+constraint=${3}
+DIR=${4}  # from root of project
+shift 4
```

Comment:
I would prefer not to have these defaults in multiple places. Since they are already set up in `tabular_benchmark.py`, I propose we remove them here. Let me know if you disagree.

Reply:
Good point, we can remove them.